app/src/main/java/org/standardnotes/notes/LoginActivity.kt

```diff
@@ -1,32 +1,34 @@
 package org.standardnotes.notes
 
-import android.content.DialogInterface
 import android.content.Intent
 import android.os.Bundle
 import android.support.v7.app.AlertDialog
 import android.support.v7.app.AppCompatActivity
+import android.text.Editable
+import android.text.TextWatcher
 import android.view.LayoutInflater
 import android.view.View
+import android.view.WindowManager
+import android.view.inputmethod.EditorInfo
+import android.widget.EditText
+import android.widget.TextView.OnEditorActionListener
 import android.widget.Toast
 import kotlinx.android.synthetic.main.activity_login.*
 import org.standardnotes.notes.comms.Crypt
 import org.standardnotes.notes.comms.data.AuthParamsResponse
 import org.standardnotes.notes.comms.data.SigninResponse
+import org.standardnotes.notes.store.ValueStore
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
-import android.widget.LinearLayout
-import android.widget.EditText
-
 
 
 /**
  * A login screen that offers login via email/password.
  */
 class LoginActivity : AppCompatActivity() {
 
-
-
+    @Override
```

Comment:
Ha, yeah I didn't even notice that.  I guess with Kotlin's override the annotation is always going to be redundant.